### PR TITLE
feat: add configuration builder

### PR DESCRIPTION
**Pull request by sdelamo**
Not sure how to tell the build plugin that there is not baseline.

To build `./gradlew build -x japiCmp`

### CODE REVIEW

**Comment by timyates** [comment]
Add

```
// Required until the first release is out
micronautBuild {
    binaryCompatibility {
        enabled = false
    }
}
```

To the build.gradle for each module

File: gradle/libs.versions.toml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[versions]
+managed-microstream='06.00.00-MS-GA'
```
Comment (timyates) [inline gradle/libs.versions.toml:2]: Should be `06.01.00-MS-GA`

File: gradle/libs.versions.toml [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+
+[libraries]
+
+microstream-storage-embedded-configuration = { module = 'one.microstream:microstream-storage-embedded-configuration', version.ref = 'managed-microstream' }
```
Comment (timyates) [inline gradle/libs.versions.toml:6]: Should be

```
managed-microstream-storage-embedded-configuration
```

**Review by timyates — CHANGES_REQUESTED** [review]

**Review by timyates — APPROVED** [review]

**Comment by timyates** [comment]
Not sure why sonarqube is failing 🤔

**Comment by sdelamo** [comment]
> Not sure why sonarqube is failing 🤔

I just added it to sonar.cloud

**Pull request merged**